.github/workflows/publish-brew-package.yml

```diff
@@ -38,6 +38,12 @@ jobs:
       - name: Checkout hazelcast-packaging repo
         uses: actions/checkout@v4
 
+      - name: Checkout hazelcast-docker repo
+        uses: actions/checkout@v4
+        with:
+          repository: hazelcast/hazelcast-docker
+          path: 'hazelcast-docker'
```

Comment:
Have mixed feeling cloning this repo. Maybe at least checkout only a single file:
```yaml
sparse-checkout: .github/scripts/abstract-simple-smoke-test.sh
sparse-checkout-cone-mode: false
```
And then maybe extract it to composite action and replace duplicates?

Reply:
> Have mixed feeling cloning this repo. Maybe at least checkout only a single file:

But `abstract-simple-smoke-test.sh` `source`s `logging.functions.sh`, so it needs that as well.
And then `hazelcast-packaging` has even tighter coupling on the implementation.

Reply:
I see, right